NAMESPACE

```diff
@@ -77,6 +78,9 @@ export(dataset_connector)
 export(dataset_connector_file)
 export(dataset_file)
 export(example_cdisc_data)
+export(formatters_var_labels)
+export(formatters_var_relabel)
+export(formatters_with_label)
```

Comment:
Why to prefix functions names by `formatters_`? `teal.data::var_labels` seems to be fine. Alternatively, we can use completely new name, for example `col_labels` or `collabels`

```suggestion
export(formatters_var_labels)
export(formatters_var_relabel)
export(formatters_with_label)
```

Reply:
Hey @gogonzo I wanted to keep the name of the function as it was, but figured out there might be a clash of NAMESPACEs if somewhere someone loads both formatters and teal.data and do not prefix functions. Keeping the name the same points directly it is the same function and prefixing with `formatters_` show the origin of the function.

I would stick with `formatters_var_labels` but can accept `col_labels`. Let me know what if your gut feeling in here and I'll proceed with the change here and in other packages affected by this PR.(on of course will do the same with `formatters_var_relabel`

Reply:
What do you think about having this long name? I don't see the sense of prefixing by `formatters_`. If prefix is to inform about origin of the code - then I think it is irrelevant. In the future function in `teal.data` or `formatters` can change and the name will be missleading. I like `col_labels`

Reply:
Good, `col_labels` it is

Reply:
a44040898d1d21d58b38878faf551aafe28b6147